packages/browser/test/client.test.ts

```diff
@@ -1,38 +1,134 @@
+import { Source } from '../src/config';
 import { ExperimentClient } from '../src/experimentClient';
-import { ExperimentUser } from '../src/types/user';
+import { ExperimentUser, ExperimentUserProvider } from '../src/types/user';
+import { Variant, Variants } from '../src/types/variant';
+
+const delay = (ms: number) => new Promise((res) => setTimeout(res, ms));
+
+class TestUserProvider implements ExperimentUserProvider {
+  getUser(): ExperimentUser {
+    return testUser;
+  }
+}
 
 const API_KEY = 'client-DvWljIjiiuqLbyjqdvBaLFfEBrAvGuA3';
```

Comment:
These are tests against the live API?

Reply:
Yes. I could mock the response eventually, but its a lot easier to hit the live endpoint.